app/controllers/concerns/course/notifications_concern.rb

```diff
@@ -0,0 +1,15 @@
+module Course::NotificationsConcern
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :load_unread_popups
+  end
+
+  private
+
+  def load_unread_popups #:nodoc:
```

Comment:
does it matter if i only load the current course? What if it is from another course? Do I want it to appear?